doc/liblouis.texi

```diff
@@ -1239,16 +1239,31 @@ repeated --- 36-36-36 shorten separator lines made with hyphens
 @end example
 
 @opcode{repword, characters dots}
-When characters are encountered check to see if the word before this
-string matches the word after it. If so, replace characters with dots
-and eliminate the second word and any word following another
-occurrence of characters that is the same. This opcode is used in
-Malaysian braille. In this case the rule is:
+When characters are encountered check to see if the word before these
+characters matches the string after them. If so, replace the
+characters with dots and eliminate the repeated string and any string
+following another occurrence of characters that is the same. This
+opcode is used in Malaysian braille. In this case the rule is:
```

Comment:
I don't understand this sentence? How about the following (inspired by [Herberts mail](https://www.freelists.org/post/liblouis-liblouisxml/Malay-braille-Code-Translation-Table-for-Malaysia,2):

``` texinfo
For some braille systems you have the requirement to remove repeated words which are connected 
by some character. In Malaysian braille for example you want to contract as follows:

@example
Text: tasik-tasik
Grade 1: 2345-1-234-24-13-36-2345-1-234-24-13
Grade 2: 2345-1-234-24-13-123456
@end example

The dash and the second occurence of 'tasik' is replaced with dots 123456. To achieve this you can use 
the repword opcode as follows:

@example
repword - 123456
@end example
```

Reply:
OK.